### PR TITLE
Ag 3371 - review getting started docs

### PR DESCRIPTION
**Pull request by cfoskin**
JIRA https://issues.jboss.org/browse/AEROGEAR-3375

Ping @finp @jstaffor

### CODE REVIEW

File: modules/ROOT/pages/_partials/generic-provisioning.adoc [hunk 1]
```diff
@@ -1,6 +1,7 @@
 To provision the {service-name} mobile service:
 
 . Log into the OpenShift console.
+. Choose your project.
```
Comment (finp) [inline modules/ROOT/pages/_partials/generic-provisioning.adoc]: doesn't it default to a project?

File: modules/ROOT/pages/con_services.adoc [hunk 2]
```diff
@@ -54,6 +54,7 @@ include::{partialsdir}/generic-service-intros.adoc[tags=device-security-service]
 include::provisioning-services.adoc[leveloffset=1]
 // include::integrating-services.adoc[leveloffset=1]
 
+include::{partialsdir}/registering-a-mobile-app.adoc[leveloffset=1]
```
Comment (finp) [inline modules/ROOT/pages/con_services.adoc]: +1

File: modules/ROOT/pages/_partials/generic-provisioning.adoc (hunk 1 above)
Comment (cfoskin) [reply modules/ROOT/pages/_partials/generic-provisioning.adoc]: It just brings you to catalog page and you have to choose your project from the list on the side navbar with list of projects.

to be able to choose "Add to project" we need to select the project first

**Comment by cfoskin** [comment]
@finp  are you happy for me to merge?

File: modules/ROOT/pages/_partials/registering-a-mobile-app.adoc [hunk 3]
```diff
@@ -0,0 +1,23 @@
+include::{partialsdir}/attributes.adoc[]
+
+= Registering a Mobile App
+
+Prerequisites:
```
Comment (josemigallas) [inline modules/ROOT/pages/_partials/registering-a-mobile-app.adoc]: Let's make this one a header as it is in other docs `== Prerequisites`

File: modules/ROOT/pages/configuring-dev-env.adoc [hunk 4]
```diff
@@ -72,6 +73,8 @@ target :'YourTarget' do
     ## Require one of the AeroGear SDK's here
     pod 'AGSPush', '{release-number}'
     pod 'AGSAuth', '{release-number}'
+    pod 'AGSCore', '{release-number}'
```
Comment (josemigallas) [inline modules/ROOT/pages/configuring-dev-env.adoc]: Rather trivial but could we keep the same order of the imports in every platform?

File: modules/ROOT/pages/_partials/registering-a-mobile-app.adoc (hunk 3 above)
Comment (cfoskin) [reply modules/ROOT/pages/_partials/registering-a-mobile-app.adoc]: 👍

File: modules/ROOT/pages/configuring-dev-env.adoc (hunk 4 above)
Comment (cfoskin) [reply modules/ROOT/pages/configuring-dev-env.adoc]: 👍

**Pull request merged**